### PR TITLE
feat: add qgis example with earthquake data

### PR DESCRIPTION
**Pull request by ruben-arts**

### CODE REVIEW

File: examples/qgis/pixi.toml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+authors = ["Ruben Arts <ruben@prefix.dev>"]
+channels = ["conda-forge"]
+platforms = ["linux-64", "win-64", "osx-64", "osx-arm64"]
+
+[tasks]
+get-data = "python get_data.py"
+start = { cmd = "qgis earthquakes_example.qgz", depends_on = "get-data"}
+
+[dependencies]
+qgis = "3.32.1.*"
```
Comment (travishathaway) [inline examples/qgis/pixi.toml:14]: You don't need `qgis` for this example. `geopandas` is enough.

**Comment by travishathaway** [comment]
Also, you should rename to `examples/geopandas`. QGIS is a GUI application 😄.

**Comment by ruben-arts** [comment]
> Also, you should rename to `examples/geopandas`. QGIS is a GUI application 😄.

Ah yeah if you run `pixi run start` it should actually start the GUI. If not please let me know.

**Review by baszalmstra — APPROVED** [review]
Works on my machine (Windows) 👍

**Pull request merged**